README.md

```diff
@@ -100,6 +100,8 @@ Reading a sequence repository requires several Python packages, all of which are
 available from pypi. Installation should be as simple as `pip install
 biocommons.seqrepo`.
 
+Acquiring SeqRepo snapshots using the CLI requires an [rsync](https://github.com/RsyncProject/rsync) binary to be available on the system $PATH. Note that [openrsync](https://www.openrsync.org/), which now ships with new MacOS installs, does not support all required functions. Mac users should install rsync from [HomeBrew](https://formulae.brew.sh/formula/rsync) and ensure that it's available on the $PATH.
```

Comment:
@jsstevenson what do you get when you run `which rsync`?

Reply:
> @jsstevenson what do you get when you run which rsync?

I wound up just installing mine via homebrew so it's in `/opt/homebrew/bin/`. In the latest MacOS version, openrsync appears to be located at `/usr/bin/rsync`

Reply:
Oh, you can also use `--rsync-exe` to point to a binary even if it's not on your $PATH, added a note about that